live-examples/css-examples/flexbox/order.html

```diff
@@ -30,10 +30,10 @@
 
 <div id="output" class="output large hidden">
   <section id="default-example" class="default-example">
-    <div id="example-element" class="transition-all">Order me</div>
-    <div style="order: 1">Order 1</div>
-    <div style="order: 2">Order 2</div>
-    <div style="order: 2">Order 2</div>
-    <div style="order: 3">Order 3</div>
+    <div id="example-element" class="transition-all">box 1:&nbsp;</div>
```

Comment:
I'd perhaps just capitalize all these instances of "box" at the start of each container. So "box" -> "Box".